integration-tests/src/test/java/tachyon/worker/block/meta/TieredStoreIntegrationTest.java

```diff
@@ -72,7 +81,7 @@ public void pinFileTest() throws Exception {
 
     // Pin the file
     mTFS.pinFile(fileId);
-    CommonUtils.sleepMs(LOG, CommonUtils.getToMasterHeartBeatIntervalMs(mWorkerConf) * 3);
+    CommonUtils.sleepMs(LOG, heartbeatInterval() * 3);
```

Comment:
why not setting `mWorkerToMasterHeartbeatIntervalMs` in `before`? It is really unusual to set a member variable in its first appearance in a test case and reference this member later on in the same test case---member variable in this case ca be replaced by a local variable.